src/config/Defaults.ts

```diff
@@ -40,6 +40,7 @@ export const DefaultConfigRoot: BridgeConfigRoot = {
         roomSetupWidget: {
             addOnInvite: false,
         },
+        allowedIpRanges: [],
```

Comment:
Wondering if we want this to actually show up as a default, because the default value would block everything currently :)